pulp_2_tests/tests/docker/api_v2/test_remove.py

```diff
@@ -12,8 +12,155 @@
 from pulp_2_tests.tests.docker.utils import get_upstream_name
 
 # Dummy Data - Need to datamine a DOCKER repo
-DOCKER_REMOVE = {'INITIAL': {'MANIFEST': 10, 'MANIFEST_LIST': 2, 'BLOB': 8, 'TAG': 13}}
+DOCKER_REMOVE = {
+    'FULL': {
+        'docker_manifest': 14,
+        'docker_manifest_list': 4,
+        'docker_blob': 8,
+        'docker_tag': 18,
+    },
+    'NON_SHARED_ML': {
+        'docker_manifest': 4,
+        'docker_manifest_list': 2,
+        'docker_blob': 8,
+        'docker_tag': 0,
+    },
+    'NON_SHARED_ML_DELETE': {
+        'docker_manifest': 2,
+        'docker_manifest_list': 1,
+        'docker_blob': 4,
+        'docker_tag': 0,
+    },
+    'SHARED_ML': {
+        'docker_manifest': 3,
+        'docker_manifest_list': 2,
+        'docker_blob': 6,
+        'docker_tag': 0,
+    },
+    'SHARED_ML_DELETE': {
+        'docker_manifest': 3,
+        'docker_manifest_list': 1,
+        'docker_blob': 6,
+        'docker_tag': 0,
+    },
+    'NON_SHARED_M': {
+        'docker_manifest': 2,
+        'docker_manifest_list': 0,
+        'docker_blob': 4,
+        'docker_tag': 0,
+    },
+    'NON_SHARED_M_DELETE': {
+        'docker_manifest': 1,
+        'docker_manifest_list': 0,
+        'docker_blob': 2,
+        'docker_tag': 0,
+    },
+    'SHARED_M_DELETE': {
+        'docker_manifest': 1,
+        'docker_manifest_list': 0,
+        'docker_blob': 3,
+        'docker_tag': 0,
+    },
+    'NONE': {
+        'docker_manifest': 0,
+        'docker_manifest_list': 0,
+        'docker_blob': 0,
+        'docker_tag': 0,
+    },
+    'ALL_V2': {
+        'docker_manifest': 5,
+        'docker_manifest_list': 4,
+        'docker_blob': 10,
+        'docker_tag': 0,
+    },
+    'ALL_V2_M_DELETE': {
+        'docker_manifest': 0,
+        'docker_manifest_list': 4,
+        'docker_blob': 0,
+        'docker_tag': 0,
+    },
+    'ALL_V2_B_DELETE': {
+        'docker_manifest': 5,
+        'docker_manifest_list': 4,
+        'docker_blob': 0,
+        'docker_tag': 0,
+    },
+    'ALL_V2_M_COPY': {
+        'docker_manifest': 5,
+        'docker_manifest_list': 0,
+        'docker_blob': 10,
+        'docker_tag': 0,
+    },
+    'ALL_V2_B_COPY': {
+        'docker_manifest': 0,
+        'docker_manifest_list': 0,
+        'docker_blob': 10,
+        'docker_tag': 0,
+    },
+}
 
+DOCKER_UNIT_TYPES = [        
+        'docker_tag',
+        'docker_manifest_list',
+        'docker_manifest',
+        'docker_blob'
+]
+
+DOCKER_UPSTREAM_NAME = 'pulp/test-fixture-1'
+
+NON_SHARED_MANIFEST_LIST = [ 
```

Comment:
I got why of those...but we should avoid to hardcode checksums. Are they going to change if we generate those fixtures again? Perhaps as long term solution we should dump those checkums in a file when generating the fixtures, and parse them, instead of hardcode them.

Reply:
Very possible. We can take the pulp-qe-tools script and incorporate that into a setupClass List/Dictionary for the run. I think that will be past the scope of this initial PR.